doc/source/getting_started/introduction.ipynb

```diff
@@ -342,7 +348,7 @@
     "\n",
     "## Geometry relations\n",
     "\n",
-    "We can also ask about the spatial relations of different geometries. Using the geometries above, we can check which of the buffered boroughs intersect the original geometry of Brooklyn, i.e., is within 10 000 feet from Brooklyn.\n",
+    "We can also ask about the spatial relationships of different geometries. Using the geometries above, we can check which of the buffered boroughs intersect the original geometry of Brooklyn, i.e., is within 10 000 feet from Brooklyn.\n",
```

Comment:
Are you sure about this change? I think that _relation_ is the correct term here. See https://en.wikipedia.org/wiki/Spatial_relation

Reply:
Agree! Spatial relation it is.